app/models/document.rb

```diff
@@ -21,9 +21,29 @@ class Document < ApplicationRecord
   belongs_to :submission
 
   def external_url
+    if ENV["RAILS_ENV"] == "production" || ENV["RAILS_ENV"] == "staging"
+      prod_external_url
+    else
+      dev_external_url
```

Comment:
instead of naming these after an environment, can we call them `s3_link` and `filesystem_link`?